src/guides/v2.2/cloud/cdn/fastly-vcl-blocking.md

```diff
@@ -0,0 +1,100 @@
+1. Expand the **Edge ACL** section.
+1. Click **Add ACL** to create a list. For this example, name the list "blocklist".
+1. Enter IP address values in the list. Any client IPs added to this list will be blocked access from the site.
+1. Optionally, select the **Negated** checkbox if needed.
+
+You will reference the Edge ACL by name in your VCL snippet code.
+
+## Create blocklist.json {#vcl}
+
+{:.bs-callout-info}
+This example shows advanced users how to create custom VCL code snippet to configure blocking rules that can be uploaded to the Fastly service using the Fastly API. You can also configure a blocklist or allowlist from the {{ site.data.var.ee }} Admin UI. See [Blocking](https://github.com/fastly/fastly-magento2/blob/master/Documentation/Guides/BLOCKING.md) in the Fastly CDN for Magento 2 module documentation.
+
+After you define the Edge ACL, you can use it to create the VCL snippet to block access to the IP addresses specified in the ACL. You can use the same VCL snippet in both Staging and Production environments, but you must upload the snippet to each environment separately.
+
+Create a `blocklist.json` file with the following VCL code in JSON format:
+
+```json
+{
+  "name": "blocklist",
+  "dynamic": "0",
+  "type": "recv",
+  "priority": "5",
+  "content": "if ( client.ip ~ blocklist) { error 403 \"Forbidden\"; }"
+}
+```
+
+Review the following parameter values and update your code snippet if necessary:
+
+-  `name`: Name for the VCL snippet. For this example, we used the name `blocklist`.
+-  `priority`: Determines the order VCL snippets call. You want to set the priority to 5 to immediately run and check for blocked IP addresses. This priority runs the snippet immediately and before any of the uploaded and default Magento VCL snippets (magentomodule) that have a priority of 50.
+-  `type`: For this VCL, we use `recv`, which places it in the `vcl_recv` subroutine, below the boilerplate VCL and above any objects.
+-  `content`: The snippet of VCL code to run, which checks the client IP address. If the IP is in the Edge ACL, it is blocked from access with a `403 Forbidden` error for the entire website. All other client IP addresses are allowed access.
+
+{:.bs-callout .bs-callout-info}
+The default VCL snippets you uploaded included a prepended name of `magentomodule_` with a priority of 50. For your custom VCL snippets, **do not use the `magentomodule_` name**. Also consider the priority of your custom snippets if they should override the default snippets.
+
+## Finish adding the VCL {#complete}
+
+After saving the VCL snippet, add the VCL snippet to the Fastly service configuration. See [Add VCL snippets using the Fastly API]({{ page.baseurl }}/cloud/cdn/cloud-vcl-custom-snippets.html#add-vcl-snippets-using-the-fastly-api).
+
+## Additional VCL examples for blocking requests
+
+The following additional examples show the VCL code to block requests without using an ACL list.
+
+**VCL code sample: Block by Country Code:**
```

Comment:
This seems strange to use two colons on same line. 
Would this be an opportunity to use an h3?

```suggestion
### VCL code sample: Block by Country Code
```